lib/jsdom/browser/Window.js

```diff
@@ -77,34 +78,35 @@ function Window(options) {
   this.clearTimeout = stopTimer.bind(this, window);
   this.__stopAllTimers = stopAllTimers.bind(this, window);
 
+  this._virtualConsole = getVirtualConsole(window);
```

Comment:
OK, now that I see how this panned out, a new suggestion to be a bit simpler: have `virtual-console.js` default-export `VirtualConsole`, and then you can just do `this._virtualConsole = new VirtualConsole()`. Then you can inline `exports.getVirtualConsole = function (window) { return window._virtualConsole; }`.